priceservice/src/app.controller.ts

```diff
@@ -13,21 +13,26 @@ export class AppController {
     this.databaseUrl = configService.get<string>("BACKEND_DB_SERVICE_URL", "http:/localhost:3000/");
   }
   /**
-   * Handles get requests to the default url of database service
+   * Handles get requests to the default url of database service. Requests are send via a circuit breaker, so timeouts and open circuit breaker erros are possible.
+   * @returns the value received from the database service or an error if request was not succesful for any reason (like timeout or CB open).
    */
   @Get()
   sendRequest() {
     return this.appService.handleRequest(this.databaseUrl);
   }
   /**
    * Handles get requests to receive balance from database service
+   * @returns  A promise with the balance received from the database service or an error if request was not succesful for any reason. 
```

Comment:
missing blank line & typo
```suggestion
   * Handles get requests to receive balance from database service
   *
   * @returns  A promise with the balance received from the database service or an error if request was not successful for any reason. 
```